django_lightweight_queue/runner.py

```diff
@@ -30,17 +30,33 @@ def handle_term(signum, stack):
         running.value = 0
     signal.signal(signal.SIGTERM, handle_term)
 
-    # Start the cron scheduler before setting up the workers so that it can
-    # populate app_settings.WORKERS
-    CronScheduler(running, log.level, log_filename_fn('cron_scheduler')).start()
+    # For the cron scheduling configuration, even if we're not running
+    # cronjobs, as it changes the queue count.
+    get_config()
+
+    if machine_number == 1:
+        # Start the cron scheduler before setting up the workers so that it can
+        # populate app_settings.WORKERS
```

Comment:
Does this populate (or need populating) when the Cron scheduler isn't running? Does the line above sort this out instead?